hack/testing/rsyslog/vars.yaml

```diff
@@ -0,0 +1,10 @@
+rsyslog__enabled: true
+# install viaq packages & config files
+rsyslog__viaq: true
+rsyslog__capabilities: [ 'viaq' ]
+rsyslog__group: root
+rsyslog__user: root
+# to share rsyslog__dir with roles/openshift_logging_rsyslog
+rsyslog__dir: /etc/rsyslog.d
+rsyslog__viaq_path: /etc/rsyslog.d/viaq
```

Comment:
I believe it should be 
rsyslog__viaq_path: '{{ rsyslog__dir }}/viaq'

Reply:
That was my first intention, but ansible did not like to have {{ ... }} in vars.yaml. :(

Reply:
Not sure why. It should work. Maybe you were missing '{{ }}' ?

Reply:
try
```
 rsyslog__viaq_path: "{{ rsyslog__dir }}/viaq"
```
note the double quotes - https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#hey-wait-a-yaml-gotcha